taxi/models.py

```diff
@@ -27,6 +28,21 @@
     def get_absolute_url(self):
         return reverse("taxi:driver-detail", kwargs={"pk": self.pk})
 
+    def clean_license_number(self):
+        if len(self.license_number) != 8:
+            raise forms.ValidationError(
+                "The license should must of 8 characters"
+            )
```

Comment:
The `ValidationError` should be imported from `django.core.exceptions`, not `django.forms`. Change `forms.ValidationError` to `ValidationError` after importing it correctly.